.github/workflows/check-publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Check publish
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  check-publish:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+    runs-on: ${{ matrix.os }}
+    container:
+      image: paritytech/ci-unified:bullseye-1.70.0-2023-05-23-v20230706
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+
+      - name: install parity-publish
+        run: cargo install --git https://github.com/paritytech/parity-publish --branch ci --profile dev
```

Comment:
Can we publish the publisher :nerd_face: ?

Reply:
We can, but as it's still early days and we're still iterating on it a lot it would be nice to use a branch so we can push updates as needed. But I can change this if needed.

Reply:
Installing from registry now.